src/style.css

```
@@ -1,3 +1,77 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
```

it's not a good practice to reset styles like this
```suggestion
body {
  margin: 0;
}
```